examples/optimal_information_transport_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Example of shape-based image reconstruction
+using optimal information transformation.
+"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+import numpy as np
+import matplotlib.pyplot as plt
+from odl.util import snr
+from odl.deform import optimal_information_transport_solver
+import odl
+standard_library.install_aliases()
+
+
+I0name = './pictures/c_highres.png'
```

Comment:
We should be rather careful in adding data to the core package since it makes ODL larger and harder to install.

But as you say. *if* we add more of these, they should be properly added to `odl.util` so that they are accessible by everyone.